contents/docs/product-analytics/web-vitals.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Web vitals autocapture
+sidebar: Docs
+showTitle: true
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import { ProductVideo } from 'components/ProductVideo'
+
+PostHog can automatically capture [core web vitals](https://web.dev/explore/learn-core-web-vitals) like LCP, FID, CLS, and FCP. This you means you don't need to manually add logging from [Google's web vitals library](https://github.com/GoogleChrome/web-vitals).
+
+## How to enable web vitals autocapture
+
+Go to your [Autocapture & heapmaps settings](https://us.posthog.com/project/settings/project-autocapture#web-vitals-autocapture). Then, in the **Web vitals autocapture** section, click **Enable**.
+
+Note that web vitals autocapture is separate from our regular [autocapture](/docs/product-analytics/autocapture) feature. Web vitals can be captured regardless of whether autocapture is enabled.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1725632828/posthog.com/contents/Screenshot_2024-09-06_at_3.26.12_PM.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/v1725632829/posthog.com/contents/Screenshot_2024-09-06_at_3.26.18_PM.png"
+  alt="How to enable web vitals autocapture" 
+  classes="rounded"
+/>
+
+Once enabled, web vitals will be captured under the `Web vitals` event name.
```

Comment:
pricing is a little confusing since you'll get 0 to 4 events per page view

I wonder if we should call out cost or not

maybe the "we capture multiple metrics per event when we can to help reduce cost" is the angle

Reply:
yes, we should mention this.

Reply:
This is clearer now that you can choose which metric to log. Will need to update the screenshot though